Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponent_Common.jinja

```diff
@@ -393,6 +397,26 @@ namespace {{ Component.attrib['Namespace'] }}
         {{ ComponentNameBase }}::Reflect(context);
     }
 
```

Comment:
Might be good to inject a comment as to why the base functions need to be called.

Reply:
Good point! I thought that's what adding the base class to the serialization classes handles
_serializeContext->Class<NetworkRigidBodyComponent, NetworkRigidBodyComponentBase>_ 
Maybe there's an equivalent for Edit and Behavior context...

Reply:
Removed that update altogether. I was mistaken that the derived HAD to call the base in order to work; I reverted the jinja derived class comment from including services at all. Most cases the services will be defined right inside the XML, otherwise, if someone really wants a custom cpp service method, they can add it and remember to call the base.